src/include/pgagroal.h

```diff
@@ -49,10 +49,10 @@ extern "C" {
 
 #define MAIN_UDS ".s.pgagroal"
 
-#ifdef HAVE_LINUX
-    #define PGAGROAL_DEFAULT_CONFIGURATION_PATH "/etc/pgagroal/"
+#ifdef HAVE_FREEBSD
+  #define PGAGROAL_DEFAULT_CONFIGURATION_PATH "/usr/local/etc/pgagroal/"
 #else
-   #define PGAGROAL_DEFAULT_CONFIGURATION_PATH "/usr/local/etc/pgagroal/"
+  #define PGAGROAL_DEFAULT_CONFIGURATION_PATH "/etc/pgagroal/"
```

Comment:
Spacing

Reply:
I've used the same indentation level as before, and removed extra spaces.